armory/datasets/config_load.py

```diff
@@ -0,0 +1,85 @@
+"""
+Temporary file for testing loading from config without modifying armory.utils
+"""
+
+import copy
+
+from armory.datasets import load, preprocessing, generator
+
+
+def actual_loader(
+    name,
+    version=None,
+    split="test",
+    shuffle_files=False,
+    framework="numpy",
+    epochs=1,
+    drop_remainder=False,
+    num_batches=None,
+    batch_size=1,
+    shuffle_elements=False,
+):
+    info, ds_dict = load.load(name, version=version, shuffle_files=shuffle_files)
+    preprocessor = preprocessing.get(name, version=version)
+    return generator.ArmoryDataGenerator(
+        info,
+        ds_dict,
+        split=split,
+        batch_size=batch_size,
+        framework=framework,
+        epochs=epochs,
+        element_filter=None,
```

Comment:
is setting `element_filter` to `None` temporary? One thing that appears to be still todo is incorporating `filter_by_class` and `filter_by_index`